schema/generate.go

```diff
@@ -347,6 +304,78 @@ func createObjectFunctionToObjectTypeMapping() map[string]string {
 	return objectFuncToObject
 }
 
+func patchMissingTypeInfo(param *Type, name, methodName string) {
+	patchToStringArray := func() {
+		if param.Items == nil {
+			param.Items = &Type{
+				Type: "string",
+			}
+		}
+	}
+	patchToIntArray := func() {
+		if param.Items == nil {
+			param.Items = &Type{
+				Type: "int",
+			}
+		}
+	}
+
+	patchToBeArray := func() {
+		if param.Type == "" {
+			param.Type = "array"
+		}
+	}
+
+	switch name {
```

Comment:
nit: This can be written more tersely, by specifying more conditions per case, like so:
```go
switch name {
case "qLabels", "qObjectIdsToPatch", etc:
  patchToStringArray()
}
```